taps/executor/parsl.py

```diff
@@ -94,7 +95,9 @@ class ParslHTExConfig(TapsExecutorConfig):
         'parsl-htex',
         description='Executor name.',
     )
-    htex: HTExConfig = Field(description='HTEx configuration.')
+    htex: HTExConfig | Dict[str, HTExConfig] = Field(  # noqa: UP006
```

Comment:
Pydantic is a bit picky about certain annotatin syntax so this will need to be written as:
```python
htex: Union[HTExConfig, Dict[str, HTExConfig]] = Field( # noqa: UP006,UP007
```